visual_behavior/data_access/from_lims.py

```diff
@@ -2214,6 +2214,53 @@ def load_objectlist(ophys_experiment_id):
     return objectlist_dataframe
 
 
+def get_well_known_file_path(id, attachable_type, asset_name):
```

Comment:
Does this work for all types of well known files?

Reply:
Second question, does the filepath it returns work for different operating systems?

Reply:
Yes, it returns works for all well known file types. For example:
```
from_lims.get_well_known_file_path(951410079, 'OphysSession', 'OphysRigSync')
```
Returns:

> '/allen/programs/braintv/production/neuralcoding/prod0/specimen_840355567/ophys_session_951410079/951410079.h5'

and 
```
from_lims.get_well_known_file_path(964440177, 'OphysExperiment', 'OphysMotionXyOffsetData')
```
Returns:

> '/allen/programs/braintv/production/visualbehavior/prod2/specimen_887127198/ophys_session_963496285/ophys_experiment_964440177/processed/964440177_suite2p_rigid_motion_transform.csv'

Note that those are the paths exactly as they are returned by LIMS. They are linux compatible, but not Windows compatible. I'll go ahead and add some code to prepend a slash and make it work for Windows.

Reply:
There is a function in utils that should do this. 

utils.correct_filepath(filepath)

Reply:
See my latest commit. It was just a matter of prepending an extra slash.